src/main/resources/assets/vampirism/lang/en_us.json

```
@@ -697,8 +697,18 @@
   "command.vampirism.argument.action.notfound": "The given action %s does not exist",
   "command.vampirism.argument.faction.notfound": "The given faction %s does not exist",
   "command.vampirism.argument.faction.notplayable": "You can not play as %s",
+  "command.vampirism.biome.notFound": "The biome %s does not exists",
```

```suggestion
  "command.vampirism.biome.not_found": "The biome %s does not exist",
```
No camel case in lang files just in case.
Also no *s because its already in `does`